pkg/spec/spec.go

```diff
@@ -606,6 +616,9 @@ func addUTSNS(config *CreateConfig, g *generate.Generator) error {
 	if utsMode.IsHost() {
 		return g.RemoveLinuxNamespace(string(spec.UTSNamespace))
 	}
+	if utsMode.IsContainer() {
+		logrus.Debug("using container utsmode")
```

Comment:
Which container is using utsmode?

Reply:
this is following the format of the other namespace parsing functions. None have access to the container ID, so it's not printed for any of them

Reply:
You do see where in a N-user/container environment this is just noise and zero value?

Reply:
I certainly do, I'll remove all of them then

Reply:
EDIT: I can fix this actually

Reply:
fixed